tests/clients/test_environment.py

```diff
@@ -46,9 +139,95 @@
     "authorId": str(ENVIRONMENT.author_id),
     "createdAt": "2018-10-03T04:20:00Z",
     "updatedAt": "2018-11-03T04:21:15Z",
+    "specification": SPECIFICATION_BODY,
+}
+
+ENVIRONMENT_CREATION_RESPONSE = Environment(
+    id=uuid.uuid4(),
+    project_id=PROJECT_ID,
+    name="Test Environment",
+    description="Environment description",
+    author_id=uuid.uuid4(),
+    created_at=None,
+    updated_at=None,
+    specification=SPECIFICATION,
+)
+
+ENVIRONMENT_CREATION_RESPONSE_BODY = {
+    "environmentId": str(ENVIRONMENT_CREATION_RESPONSE.id),
+    "projectId": str(PROJECT_ID),
+    "name": ENVIRONMENT_CREATION_RESPONSE.name,
+    "description": ENVIRONMENT_CREATION_RESPONSE.description,
+    "authorId": str(ENVIRONMENT_CREATION_RESPONSE.author_id),
+    "specification": SPECIFICATION_BODY,
 }
 
 
+def test_semver_schema():
+    data = SemverSchema().load(VERSION_BODY)
+    assert data == VERSION
+
+
+def test_python_package_schema():
+    data = PythonPackageSchema().load(PYTHON_PACKAGE_BODY)
+    assert data == PYTHON_PACKAGE
+
+    data = PythonPackageSchema().load(PYTHON_PACKAGE_BODY_LATEST)
+    assert data == PYTHON_PACKAGE_LATEST
+
+
+def test_pip_schema():
+    data = PipSchema().load(PIP_BODY)
+    assert data == PIP
+
+
+def test_conda_schema():
+    data = CondaSchema().load(CONDA_BODY)
+    assert data == CONDA
+
+
+def test_python_specification_schema():
+    data = PythonSpecificationSchema().load(PYTHON_SPECIFICATION_BODY)
+    assert data == PYTHON_SPECIFICATION
+
+
+def test_apt_package_schema():
+    data = AptPackageSchema().load(APT_PACKAGE_BODY)
+    assert data == APT_PACKAGE
+
+
+def test_apt_schema():
+    data = AptSchema().load(APT_BODY)
+    assert data == APT
+
+
+def test_python_schema():
+    data = PythonSchema().load(PYTHON_BODY)
+    assert data == PYTHON
+
+
+def test_script_schema():
+    data = ScriptSchema().load(SCRIPT_BODY)
+    assert data == SCRIPT
+
+
+def test_specification_schema():
```

Comment:
The `SpecificationSchema` (and all the schemas it depends on) is used on both `dump` and `load` - we should add tests for the `dump` case as well.

I suggest that for cases where we test both directions, we should rename the existing tests like:

```suggestion
def test_specification_schema_load():
```

and add additional tests named like `test_specification_schema_dump`.